tests/test_interface.rb

```diff
@@ -677,7 +717,7 @@ def test_interface_ipv4_addr_mask_set_address_invalid
     assert_raises(RuntimeError) do
       interface.ipv4_addr_mask_set('', 14)
     end
-    interface.switchport_mode = :access
+    # interface.switchport_mode = :access
```

Comment:
I think so. It seems unnecessary (as NXAPI test still passes). Will remove in next diff.